awscrt/io.py

```diff
@@ -83,6 +94,31 @@ def on_shutdown():
         self.shutdown_event = shutdown_event
         self._binding = _awscrt.event_loop_group_new(num_threads, is_pinned, cpu_group, on_shutdown)
 
+    @staticmethod
+    def get_or_create_static_default():
+        global _static_event_loop_group
+        if '_static_event_loop_group' in globals():
+            if isinstance(_static_event_loop_group, EventLoopGroup):
+                return _static_event_loop_group
+
+        os_cpu_count = os.cpu_count()
+        if os_cpu_count is None:
+            os_cpu_count = 1  # could not find the processor count - default to 1
+        else:
+            # os.cpu_count returns all cores on the system (including virtual ones)
+            # so we divide by two to avoid using all cores on the entire system.
+            os_cpu_count = max(1, int(os_cpu_count / 2))
```

Comment:
Just create [`EventLoopGroup()`](https://github.com/awslabs/aws-crt-python/blob/cc651452fa1900c140fedde0be65962b5b897396/awscrt/io.py#L65) with no args to get default settings. Don't copy/paste the finicky default-picking logic here. If we modified the default-picking logic in C to be smarter, then this would override it